docs/cuopt/source/cuopt-server/examples/routing/examples/initial_solution_example.py

```
@@ -0,0 +1,124 @@
+    - cuOpt server running (default: localhost:5000)
+    - cuopt_sh_client package installed
+
+Note:
+    Initial solutions may not always be accepted. For very small/simple problems,
+    the solver might find the optimal solution immediately without needing the
+    initial solution.
+"""
+
+from cuopt_sh_client import CuOptServiceSelfHostClient
+import json
+import time
```

Flake8 (7.3.0)

[error] 'time' imported but unused

(F401)